src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +45,34 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        SimpleDateFormat sdf = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
sdf - I can't understant what is this by name

Reply:
done.